packages/visual-editor/src/components/Dropzone/EditorBlockClone.tsx

```diff
@@ -53,6 +53,10 @@ export const EditorBlockClone: React.FC<EditorBlockCloneProps> = ({
   const isAssemblyBlock = node.type === ASSEMBLY_BLOCK_NODE_TYPE;
   const isSingleColumn = node.data.blockId === CONTENTFUL_COMPONENTS.singleColumn.id;
 
+  if (isComponentMissing || !elementToRender) {
```

Comment:
Isn't `isComponentMissing` and `!elementToRender` sort of the same thing aliased twice? Maybe we could just use one of them?

Reply:
✅  FIXED! Thanks for noticing!